external-crates/move/crates/move-analyzer/src/inlay_hints.rs

```diff
@@ -74,6 +35,45 @@ pub fn on_inlay_hint_request(context: &Context, request: &Request, symbols: &Sym
     }
 }
 
+fn inlay_hints(context: &Context, fpath: PathBuf) -> Option<Vec<InlayHint>> {
+    let symbols_map = &context.symbols.lock().unwrap();
```

Comment:
```suggestion
    let symbols_map = &context.symbols.lock()?;
```
Or if `lock()` is a result, 
```suggestion
    let symbols_map = &context.symbols.lock().ok()?;
```